### PR TITLE
feat: Complete fizzbuzz

### PR DESCRIPTION
**Pull request by ulgoon**
solve #1

### CODE REVIEW

File: fizzbuzz.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+for i in range(1,20):
```
Comment (kingwangzzang1234) [inline fizzbuzz.py:1]: It's not pythonic

File: fizzbuzz.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    if i%15==0:
```
Comment (kingwangzzang1234) [inline fizzbuzz.py:2]: Do fizzbuzz with 1 if statement

**Review by kingwangzzang1234 — CHANGES_REQUESTED** [review]
DO this RIGHT AWAY!!!!!!!!!

File: fizzbuzz.py (hunk 1 above)
Comment (ulgoon) [reply fizzbuzz.py:1]: Yes..

File: fizzbuzz.py (hunk 2 above)
Comment (ulgoon) [reply fizzbuzz.py:2]: I'll do it

**Review by kingwangzzang1234 — APPROVED** [review]
아니.. 할 줄 아시면서 지금..

**Pull request merged**